src/final2/subtests/PraktomatPublicTest.java

```diff
@@ -12,117 +12,113 @@
 /**
  * Simulates the Praktomat's public test.
  * 
- * @author Martin L�per
+ * @author Martin L�per
  * @version 1.0
  */
 public class PraktomatPublicTest extends LangtonSubtest {
-    

```

Comment:
I only applied formatting here and fixed a typo. Once again, formatting breaks `git`